modules/gapi/src/backends/fluid/gfluidbackend.cpp

```diff
@@ -1418,7 +1418,7 @@ void cv::gimpl::GParallelFluidExecutable::run(std::vector<InObj>  &&input_objs,
 
 // FIXME: these passes operate on graph global level!!!
 // Need to fix this for heterogeneous (island-based) processing
-void GFluidBackendImpl::addBackendPasses(ade::ExecutionEngineSetupContext &ectx)
+void GFluidBackendImpl::addMetaSensitiveBackendPasses(ade::ExecutionEngineSetupContext &ectx)
```

Comment:
@rgarnov can you please review this change? The idea is to run meta-dependent passes only when meta is available.

Reply:
I believe it should not break anything in fluid